.pre-commit-config.yaml

```diff
@@ -34,14 +34,6 @@ repos:
         types: [python]
         require_serial: true
 
-      - id: pretty-format-toml
```

Comment:
Was it duplicate with `prettier`?

Reply:
It's explained in the commit message:

> - Disable pretty-format-toml incompatible with latest Poetry lock format

Reply:
Ah, my bad, I haven't seen the `...` next to the first line of your commit message to show the following lines!

Reply:
Yeah, GitHub have done a good job of making commit messages much less useful.